spec/models/cat_spec.rb

```diff
@@ -1,5 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  it "should validate name" do
+    cat = Cat.create
+    expect(cat.errors[:name]).to_not be_empty
+  end
+
+  it "should validate age" do
+   cat = Cat.create
+   expect(cat.errors[:age]).to_not be_empty
+  end
+
+  it "should validate enjoys" do
+  cat = Cat.create
+  expect(cat.errors[:enjoys]).to_not be_empty
+  end
+
+  it "should validate enjoys will have ten or more characters" do
+  cat = Cat.create
+  expect(cat.errors[:enjoys]).to_not be_empty
+  end
+
+  it "should validate image" do
+  cat = Cat.create
+  expect(cat.errors[:image]).to_not be_empty
+  end
+
```

Comment:
Make sure you are indenting your code blocks.

Reply:
Will be more diligent.